itest/lnd_routing_test.go

```diff
@@ -746,6 +748,162 @@ func testInvoiceRoutingHints(ht *lntest.HarnessTest) {
 	ht.ForceCloseChannel(alice, chanPointEve)
 }
 
+// testScidAliasRoutingHints tests that dynamically created aliases via the RPC
+// are properly used when routing.
+func testScidAliasRoutingHints(ht *lntest.HarnessTest) {
+	const chanAmt = btcutil.Amount(100000)
+
+	// Option-scid-alias is opt-in, as is anchors.
+	scidAliasArgs := []string{
+		"--protocol.option-scid-alias",
+		"--protocol.anchors",
+	}
+
+	carol := ht.NewNode("Carol", scidAliasArgs)
+	dave := ht.NewNode("Dave", scidAliasArgs)
+
+	ht.FundCoins(btcutil.SatoshiPerBitcoin, carol)
+	ht.FundCoins(btcutil.SatoshiPerBitcoin, dave)
+
+	ht.ConnectNodes(carol, dave)
+
+	// Create a channel between Carol and Dave, which uses the scid alias
+	// feature.
+	chanPointCD := ht.OpenChannel(
```

Comment:
@guggero your comments are addressed

this line here hangs and I am unsure why, still investigating

Reply:
I think this is the issue (from Carol's log):
```
unable to request alias: channel link not found
```